### PR TITLE
Replaced legacy coala-format

### PR DESCRIPTION
**Pull request by alexandruionascu**
Replaced coala-format with coala --format
agrument and upgraded atom-linter package version.

### CODE REVIEW

**Comment by sils** [comment]
coala.io/commit

File: lib/main.coffee [hunk 1]
```diff
@@ -48,6 +48,7 @@ module.exports =
       lint: (textEditor) =>
         filePath = textEditor.getPath()
         parameters = []
+        parameters.push '--format'
```
Comment (sils) [inline lib/main.coffee]: asd

**Comment by Mixih** [comment]
You need tos et ownership in you local git program. This can be done with `git config --global user.email "your_github_email_"`. After that, amend your commit (with `git commit -a --amend`) and force push your branch.

**Comment by Mixih** [comment]
process/wip

**Comment by jayvdb** [comment]
Please squash your three commits into one commit.

**Pull request closed without merging**